ui/pages/users/add.js

```diff
@@ -0,0 +1,162 @@
+  return (
+    <div className='flex flex-col pt-8 px-1 border rounded-lg border-gray-950'>
+      <div className='flex flex-row space-x-2 items-center px-4'>
+        <img src='/users.svg' className='w-6 h-6' />
+        <div>
+          <h1 className='text-2xs'>Add User</h1>
+        </div>
+      </div>
+      <div className='flex flex-col mt-6 mx-4 space-y-1'>
+        <div className='mt-4'>
+          <label className='text-3xs text-gray-300 uppercase'>User Email</label>
+          <input
+            autoFocus
+            spellCheck='false'
+            type='email'
+            placeholder='enter the user email here'
+            value={email}
+            onChange={onChange}
+            onKeyDown={onKeyDown}
+            className={`w-full bg-transparent border-b border-gray-950 text-3xs px-px py-3 focus:outline-none focus:border-b focus:border-gray-200 placeholder:italic ${error ? 'border-pink-500' : 'border-gray-800'}`}
+          />
+        </div>
+        {error && <ErrorMessage message={error} />}
+      </div>
+      <div className='flex flex-row justify-between m-6 items-center'>
+        <Link href='/users'>
+          <a className='uppercase border-0 hover:text-white text-gray-300 text-4xs'>Cancel</a>
+        </Link>
+        <button
+          type='button'
+          onClick={onAddUser}
+          disabled={!email}
+          className='bg-gradient-to-tr from-indigo-300 to-pink-100 hover:from-indigo-200 hover:to-pink-50 rounded-md p-0.5 text-center disabled:opacity-30'
+        >
+          <div className='bg-black rounded-md tracking-tight text-2xs px-6 py-3 text-pink-200'>
+            Add User
+          </div>
+        </button>
+      </div>
+    </div>
+  )
+}
+
+function UserOneTimePassword ({ password, onAddUser }) {
+  return (
+    <div className='flex flex-col pt-8 px-1 border rounded-lg border-gray-950'>
+      <div className='flex flex-row space-x-2 items-center px-4'>
+        <img src='/users.svg' className='w-6 h-6' />
+        <div>
+          <h1 className='text-2xs'>Add User</h1>
+        </div>
+      </div>
+      <h2 className='text-2xs pt-5 px-4'>User added. Send the user this one time password for their initial login. This password will not be shown again.</h2>
+      <div className='flex flex-col mt-6 mx-4 space-y-1'>
+        <label className='text-3xs text-gray-300 uppercase'>One Time Password</label>
+        <input
+          readOnly
+          value={password}
+          className='w-full bg-transparent text-3xs px-px py-3 focus:outline-none font-mono'
+        />
+      </div>
+      <div className='flex flex-row justify-between m-6 items-center'>
+        <button onClick={onAddUser} className='uppercase border-0 hover:text-white text-gray-300 text-4xs'>Add Another</button>
+        <Link href='/users'>
+          <a className='bg-gradient-to-tr from-indigo-300 to-pink-100 hover:from-indigo-200 hover:to-pink-50 rounded-md p-0.5 text-center disabled:opacity-30'>
```

Comment:
Nit: I would consider a single color border for now and we can reintroduce a tailwind style for this since these current "double div" borders don't have consistent border widths:

```
<Link href='/destinations'>
  <a
    className='flex-none border border-violet-300 rounded-md text-violet-100 ...'
    disabled={submitted}
  >
    Finish
  </a>
</Link>
```